### PR TITLE
Eng 6934 data contracts conditional

### PR DESCRIPTION
**Pull request by adamglasser**
## Related tasks
- Resolves ENG-6934 ENG-6798 ENG-6916

## Description
_What does this pull-request fix/improve/add?_

### CODE REVIEW

File: pipelines/directory_read/pipelines/merge_benepass/metadata.yaml [hunk 1]
```diff
@@ -10,11 +10,11 @@ blocks:
   has_callback: false
   language: python
   name: read_benepass_merge_source_data
-  status: executed
+  status: failed
```
Comment (rodolfolottin) [inline pipelines/directory_read/pipelines/merge_benepass/metadata.yaml:13]: I believe you will need to ignore the changes made on this file.

Comment (rodolfolottin) [reply pipelines/directory_read/pipelines/merge_benepass/metadata.yaml:13]: Same with `mage-ai.db`

Comment (adamglasser) [reply pipelines/directory_read/pipelines/merge_benepass/metadata.yaml:13]: done

Comment (adamglasser) [reply pipelines/directory_read/pipelines/merge_benepass/metadata.yaml]: @rodolfolottin I am not 100% sure this worked [](edited-with-Axolo)

Comment (adamglasser) [reply pipelines/directory_read/pipelines/merge_benepass/metadata.yaml]:
```
#mage stuff
./pipelines/mage_data/directory_read/mage-ai.db
*/metadata.yaml```
 [](edited-with-Axolo)

Comment (rodolfolottin) [reply pipelines/directory_read/pipelines/merge_benepass/metadata.yaml]: Try updating it to:

```
mage-ai.db
metadata.yaml
```
 [](edited-with-Axolo)

File: data-contracts/roster.json [hunk 2]
```diff
@@ -202,6 +197,70 @@
...
+      "source_key": "has_elections",
+      "source_operator": "IS_IN",
+      "source_values": [
+        "TRUE"
+      ],
+      "target_key": "elections",
+      "target_operator": "IS_NOT_IN",
+      "target_values": [
+        ""
+      ],
```
Comment (henriquesalvaro) [inline data-contracts/roster.json:262]: :chefskiss:

**Review by rodolfolottin — APPROVED** [review]

**Pull request merged**